Widgets/Widgets/SearchWidget.swift

```diff
@@ -137,7 +154,47 @@ struct SearchWidgetView: View {
             .widgetURL(entry.url)
             .background(Color(theme.colors.paperBackground))
         }
-        
+    }
+    
+    @ViewBuilder
+    private var accessoryCircularView: some View {
+        ZStack {
+            
+            Image("W")
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(width: 28, height: 28)
+                
+        }
+        .widgetURL(entry.url)
+    }
+    
+    @ViewBuilder
+    private var accessoryRectangularView: some View {
+        HStack(spacing: 8) {
+            Image("W")
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(width: 25, height: 25)
+            
+            Text(SearchWidget.LocalizedStrings.searchWikipedia)
+                .font(.system(size: 13, weight: .medium))
+                .lineLimit(1)
+
+        }
+        .widgetURL(entry.url)
+    }
+    
+    @ViewBuilder
+    private var accessoryInlineView: some View {
+        HStack(spacing: 6) {
+            Image(systemName: "magnifyingglass")
+                .font(.system(size: 11, weight: .medium))
+            
+            Text(SearchWidget.LocalizedStrings.searchWikipedia)
+                .font(.system(size: 13, weight: .medium))
+        }
+        .widgetURL(entry.url)
```

Comment:
The widget is not displaying correctly - it also needs to adopt the same solution that was done for the home screen widgets

![Screenshot 2025-06-17 at 11 32 03](https://github.com/user-attachments/assets/301382e5-c37e-4be1-8dac-f46a20df153c)